externs/lcevc.js

```diff
@@ -1,25 +1,25 @@
 /**
-* @fileoverview Externs for LcevcDil
+* @fileoverview Externs for LcevcDec
 * compiler.
 *
 * @externs
 */
 
-// This empty namespace is declared to check if LcevcDil libraries are loaded.
+// This empty namespace is declared to check if LcevcDec libraries are loaded.
 var libDPIModule = {};
-var LcevcDil = {};
+var LCEVCdec = {};
```

Comment:
This is technically a breaking change AFAICT.  Apps that upgrade Shaka to a version with this PR will also have to be modified themselves to switch to the new module.  Unless you can provide some backward compatibility, this PR would require a major version bump in Shaka Player (`feat!:` in conventional commits syntax instead of `feat:`).

I'm not ready to merge this yet, as a major version bump triggers a year-long LTS release according to our maintenance policy.

Should we defer this?  Or would you like to try making it backward compatible?

Reply:
The config parameters are not changed, hence the upgrade should not cause any issues in terms of setting up the player or the enhancement libraries. From a user perspective the only thing that would change is the library. Can you please provide more details about how we can make this backwards compatible? I am probably missing something.

Reply:
An application that includes the old DIL module will suddenly stop getting LCEVC support when this is merged.  That makes it a breaking change.

Reply:
Got it, so having functionality within the `lcevc_dec.js`  to add support to the old DIL module as the new Dec module in the `externs` would suffice right?
Can you please suggest if this is a valid approach to support the old library with the new implementation. Thanks!

Reply:
I'm sorry, I don't understand what you are proposing.  Let me try to explain backward compatibility in another way:

The standard for backward compatibility is that an application loading the old DIL module with a new Shaka Player should still work and get LCEVC support.  For Shaka Player to be backward compatible, it must continue to support the old thing.  Adding a new thing is fine, but removing an old thing is a breaking change.

Does that make sense?

Reply:
I have a latest commit addressing this issue, changes are made in the `lcevc_dec.js` to check for the availability of either `lcevc_dec.min.js` or `lcevc_dil.min.js`.  I have also added checks to the functions that are not supported. Please take a look and let me know if that addresses the issue. Thanks!